cdc/owner/feed_state_manager.go

```diff
@@ -257,26 +257,40 @@ func (m *feedStateManager) errorsReportedByProcessors() []*model.RunningError {
 	return result
 }
 
-func (m *feedStateManager) HandleError(errs ...*model.RunningError) {
+func (m *feedStateManager) handleError(errs ...*model.RunningError) {
+	// if one of the error stored by changefeed state(error in the last tick)
+	// is a fast-fail error, the changefeed should be failed
+	if m.state.Info.HasFastFailError() {
+		m.shouldBeRunning = false
+		m.patchState(model.StateFailed)
+		return
+	}
```

Comment:
Can we remove this logic, since `info.Error` is always set in this function, and checked at L272

Reply:
yeh， good advice.